.github/workflows/chroma-test.yml

```diff
@@ -19,6 +19,7 @@ jobs:
         platform: [ubuntu-latest, windows-latest]
         testfile: ["--ignore-glob 'chromadb/test/property/*' --ignore-glob 'chromadb/test/stress/*'",
                    "chromadb/test/property/test_add.py",
+                   "--ignore-glob 'chromadb/test/auth/*'",
```

Comment:
Why add this? This runs all tests not in auth, but auth tests were already run by 

--ignore-glob 'chromadb/test/property/*'

Reply:
Removed.